cmd/cli/cmd/alpha/content/content.go

```diff
@@ -0,0 +1,21 @@
+package content
+
+import "github.com/spf13/cobra"
+
+var manifestOutputDirectory string
+
+// NewCmd returns a cobra.Command for content generation operations.
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "content",
```

Comment:
I think we should put "generator/gen/generate" somewhere in the command name. Maybe `capact alpha manifest-gen ...`?